disease_gene/disease_associates_gene/dataset_statistics/dataset_statistics.py

```diff
@@ -0,0 +1,491 @@
+
+# coding: utf-8
+
+# # Dataset Statistics for Disease Gene Sentences
+
+# This notebook is designed to show statistics on the data extracted from pubmed. The following cells below here are needed to set up the environment.
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+get_ipython().run_line_magic('matplotlib', 'inline')
+
+from collections import Counter
+from itertools import product
+import os
+import pickle
+import sys
+
+sys.path.append(os.path.abspath('../../../modules'))
+
+import pandas as pd
+import matplotlib.pyplot as plt
+from matplotlib_venn import venn2
+import seaborn as sns
+from tqdm import tqdm_notebook
+
+sns.set(rc={'figure.figsize':(12,6), "font.size":17})
+
+
+# In[2]:
+
+
+#Set up the environment
+username = "danich1"
+password = "snorkel"
+dbname = "pubmeddb"
```

Comment:
It is possible to throw into a file, but it would clash with how I routinely set up my notebook. I think it would be easier to leave it how it is. If necessary to change let me know.